### ISSUE TITLE
Error Loading pyRevit due to pyRevitLabs.Emojis

### ISSUE THREAD

**Issue by Thomas-Lecuppre** [eirannejad/pyRevit#1695]
**Describe the bug**
Hi, I was using Revit 2022 and after a restart of Revit pyRevit won't load. I tried to update but nothing. I unsintall it, restarted my computer, install pyRevit 4.8.12.22247 but nothing load again.

`pyrevit attach master 277 --installed`

**To Reproduce**
1. Start Revit 2022

**Expected behavior**
I really don't know what happen. I just updated my graphics card driver.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: win 11
 - pyRevit Version 4.8.12.22247 
 - pyRevit Environment: 

`==> Registered Clones (full git repos)
==> Registered Clones (deployed from archive/image)
master | Deploy: "basepublic" | Branch: "master" | Version: "4.8.12.22247+0031" | Path: "C:\Users\ThomasLECUPPRE(Letit\AppData\Roaming\pyRevit-Master"
==> Attachments
master | Product: "Autodesk Revit 2022" | Engine: IPY277 (277) | Path: "C:\Users\ThomasLECUPPRE(Letit\AppData\Roaming\pyRevit-Master"
master | Product: "Autodesk Revit 2021" | Engine: IPY277 (277) | Path: "C:\Users\ThomasLECUPPRE(Letit\AppData\Roaming\pyRevit-Master"
master | Product: "Autodesk Revit 2020" | Engine: IPY277 (277) | Path: "C:\Users\ThomasLECUPPRE(Letit\AppData\Roaming\pyRevit-Master"
master | Product: "2019.2.6" | Engine: IPY277 (277) | Path: "C:\Users\ThomasLECUPPRE(Letit\AppData\Roaming\pyRevit-Master"
master | Product: "2018.3.3 Security Fix" | Engine: IPY277 (277) | Path: "C:\Users\ThomasLECUPPRE(Letit\AppData\Roaming\pyRevit-Master"
==> Installed Extensions
==> Default Extension Search Path
C:\Users\ThomasLECUPPRE(Letit\AppData\Roaming\pyRevit\Extensions
==> Extension Search Paths
==> Extension Sources - Default
https://github.com/eirannejad/pyRevit/raw/master/extensions/extensions.json
==> Extension Sources - Additional
==> Installed Revits
Autodesk Revit 2022 | Version: 22.1.30.34 | Build: 20220520_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2022\"
Autodesk Revit 2021 | Version: 21.1.70.21 | Build: 20220517_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2021\"
Autodesk Revit 2020 | Version: 20.2.90.12 | Build: 20220517_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2020\"
2019.2.6 | Version: 19.2.60.3 | Build: 20220224_1700(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2019\"
2018.3.3 Security Fix | Version: 18.3.3.18 | Build: 20190510_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2018\"
==> Running Revit Instances
Error: La référence d'objet n'est pas définie à une instance d'un objet.
Run with "--debug" option to see debug messages`

With debug option : https://paste.me/paste/f861f5a8-dbe7-4fef-5536-4dff7edadae1#0df78046a2bfa131f31fc9747f3f378e40bccc323bb737fef42cd1dd7a3f033c

**Comment by Thomas-Lecuppre** [comment]
It's seem that my plugin broke your. I don't know why.

**Comment by eirannejad** [comment]
Yes. Please make sure to return null for any assembly matching `pyRevitLabs.*`. Otherwise pyRevit runtime won't be able to find its own dlls e.g. `pyRevitLabs.Emojis.dll`
